### ISSUE TITLE
Enhancement : Backward compatibility in ScratchPad

### ISSUE THREAD

**Issue by natanraj** [eclipsesource/tabris-js#1228]
Will we get a chance to use scratchpad with backward compatibility ? Currently I am using 2.beta1 version for app building - but in dev app it is very latest. So with my personal plugins included in Debug app, i couldn't do any scratchpad testing.

**Comment by mpost** [comment]
@natanraj Moving forward we will stabilize the scratchpad but for the current 2.x development stream we will use the latest version.

**Comment by ralfstx** [comment]
I agree. 2.0 will soon be API complete and then the scratchpad will be stable again. Sorry for the inconvenience!

**Comment by ralfstx** [comment]
We have plans to let the *Developer App* determine the tabris version used for the scratchpad, not the *Server*.
